tools/cldr-code/src/main/java/org/unicode/cldr/tool/CLDRModify.java

```diff
@@ -2203,6 +2186,179 @@ public void handlePath(String xpath) {
+                        System.out.println("wantRevertToBaseline: return for 3");
+                    }
+                    return false;
+                }
+                if (deb) {
+                    System.out.println("wantRevertToBaseline: return true");
+                }
+                return true;
+            }
+
+            private boolean changesWereAllowed(String localeID, String xpath, String fullXPath) {
+                boolean isError = pathHasError(localeID, xpath);
+                String oldValue = baselineFileUnresolved.getWinningValue(xpath);
+                boolean isMissing = (oldValue == null || CLDRFile.DraftStatus.forXpath(fullXPath).ordinal() <= CLDRFile.DraftStatus.provisional.ordinal());
+                String locOrAncestor = localeID;
+                while (!"root".equals(locOrAncestor)) {
+                    if (SubmissionLocales.allowEvenIfLimited(locOrAncestor, xpath, isError, isMissing)) {
+                        return true;
+                    }
+                    locOrAncestor = LocaleIDParser.getParent(locOrAncestor);
+                }
+                return false;
+            }
+
+            /**
+             * These were derived from all errors found running this command:
+             * java -DCLDR_DIR=$(pwd) -jar tools/cldr-code/target/cldr-code.jar check -S common,seed -e -z FINAL_TESTING
+             * >> org.unicode.cldr.test.ConsoleCheckCLDR
+             *
+             * TODO: this is incomplete? Should include some "errors" that are not in personNames??
+             */
+            private final String[] ERR_LOCALES_PATHS = new String[] {
+                "ja", "//ldml/personNames/sampleName[@item=\"foreignFull\"]/nameField[@type=\"surname-prefix\"]",
+                "nl_BE", "//ldml/personNames/sampleName[@item=\"nativeFull\"]/nameField[@type=\"surname\"]",
+                "yue", "//ldml/personNames/sampleName[@item=\"foreignFull\"]/nameField[@type=\"title\"]",
+                "yue", "//ldml/personNames/sampleName[@item=\"foreignFull\"]/nameField[@type=\"surname-prefix\"]",
+                "yue", "//ldml/personNames/sampleName[@item=\"foreignFull\"]/nameField[@type=\"surname-core\"]",
+                "zh", "//ldml/personNames/sampleName[@item=\"foreignFull\"]/nameField[@type=\"title\"]",
+                "zh", "//ldml/personNames/sampleName[@item=\"foreignFull\"]/nameField[@type=\"surname-prefix\"]",
+                "zh", "//ldml/personNames/sampleName[@item=\"foreignFull\"]/nameField[@type=\"surname-core\"]",
+                "zh_Hant", "//ldml/personNames/sampleName[@item=\"foreignFull\"]/nameField[@type=\"title\"]",
+                "zh_Hant", "//ldml/personNames/sampleName[@item=\"foreignFull\"]/nameField[@type=\"surname-prefix\"]",
+                "zh_Hant", "//ldml/personNames/sampleName[@item=\"foreignFull\"]/nameField[@type=\"surname-core\"]",
```

Comment:
all of these should already be allowed by `SubmissionLocales.allowEvenIfLimited` which matches `personNames/.*` 

so none of these errors should have any effect.

Reply:
but harmless, right?

and there were other errors, that need to be added here?

Reply:
should be harmless

I don't know of any errors -there weren't any in baseline that i saw.

Reply:
@macchiati I think you said there were more errors in baseline? these are all I found, running console check at the command line